src/monthly.js

```diff
@@ -36,33 +67,4 @@ function processLine (line, columnHeaders) {
   return new Promise(function (resolve, reject) {
     resolve()
   })
```

Comment:
here, you can just `return Promise.resolve()` or maybe returning a promise is not necessary anymore